eng/pipelines/docindex.yml

```diff
@@ -49,13 +49,30 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml
         parameters:
           DailyBranchVariableName: DailyDocsBranchName
+      # Sync docs fork repo onboarding files/folders
+      - pwsh: |
+          $dailyDocsRepoLocation = "$(DocRepoLocation)/dailyDocs"
+          echo "##vso[task.setvariable variable=DailyDocsRepoLocation]$dailyDocsRepoLocation"
+          if (!(Test-Path $dailyDocsRepoLocation)) {
```

Comment:
Do we need to create this? Won't the clone create it?

Reply:
The working directory contains two clones, 1 main branch from Azure repo, 2 daily branch from azure-sdk repo.
1 used in schedule run updating the Azure main repo. (For release ones)
2 used in schedule run updating the Azure daily repo. (For daily docs)

Both branches get updated in schedule run.

Reply:
We can consider using the same clone but just adding a new remote that would save on some of the forking going on and the duplicate cloning.